### PR TITLE
Complete bind tasks

### PR DESCRIPTION
**Pull request by SofiaTernovskaya**

### CODE REVIEW

File: src/bind/task1/task1-tests.spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const bind = require('./task1');
+
+const chai = require('chai');
+chai.should();
+
+describe('Should do anything bind can do', function() {
+    it('Should set the value of the this to a specific object', function() {
+        var user = {
+            sayHi: function(who) {
+                return ("Привет, " + who);
```
Comment (valerybugakov) [inline src/bind/task1/task1-tests.spec.js]: To test that `this` context is correct it should be used in `sayHi` method, i.e. `return this.greeting + who;`
From this test it's not clear that context is set to `user`.

File: src/bind/task1/task1-tests.spec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const bind = require('./task1');
+
+const chai = require('chai');
+chai.should();
+
+describe('Should do anything bind can do', function() {
+    it('Should set the value of the this to a specific object', function() {
+        var user = {
```
Comment (valerybugakov) [inline src/bind/task1/task1-tests.spec.js]: `var` is inconsistent with the code style 😜
`let` is used everywhere else

File: src/bind/task1/task1.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+function bind() {
+    let args = [].slice.call(arguments, 2);
```
Comment (valerybugakov) [inline src/bind/task1/task1.js]: Maybe enhanced by more descriptive naming `let boundArgs`

File: src/bind/task1/task1.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    let func = arguments[0];
+    let context = arguments[1];
+    
+    return function() {
+        let argsNew = [].slice.call(arguments);
```
Comment (valerybugakov) [inline src/bind/task1/task1.js]: `let callArgs`

File: src/bind/task1/task1-tests.spec.js (hunk 2 above)
Comment (SofiaTernovskaya) [reply src/bind/task1/task1-tests.spec.js]: aren't you a wet blanket 
(I know you're right, I copied that part and didn't check)


(uuuuu fucker)

**Pull request merged**